javascript/javascript3/week2/Tasks/script.js

```diff
@@ -0,0 +1,41 @@
+
+//Promise that resolves after set time
+function resolveAfterSeconds(resolveAfter){
+    return new Promise((resolve)=>{
+        setTimeout(()=>{
+            resolve();
+        },resolveAfter*1000)
+    })
+}
+
+resolveAfterSeconds(3).then(()=>{
+    console.log("Resolved");
+});
+
+//Fetching and waiting
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+
+const promiseFetch = new Promise((resolve, reject)=>{
+    try{
```

Comment:
Using `try/catch` with Promises doesn't work as you think.
It will only catch errors that happen while you create the Promise, not errors that happen while the Promise is executing.
That's why you have the `.catch` handler on Promises.